worker/worker.go

```diff
@@ -78,7 +78,7 @@ func processCampaign(c *models.Campaign) {
 			t,
 			c.URL + "?rid=" + t.RId,
 			c.URL + "/track?rid=" + t.RId,
-			"<img src='" + c.URL + "/track?rid=" + t.RId + "'/>",
+			"<img style="display: none" src='" + c.URL + "/track?rid=" + t.RId + "'/>",
```

Comment:
I think single quotes are needed for the style attribute instead of double quotes: `"<img style='display: none' src='" + c.URL + "/track?rid=" + t.RId + "'/>",`